components/02-components/breadcrumbs/README.md

```diff
@@ -1 +1,5 @@
-*There are no notes for this component yet.* 
+The breadcrumbs component is used for navigation and to give the user awareness of their location within the site structure.
+
+The background container color for the breadcrumbs bar should match the color assigned to the information architecture of that section of the website. You can find this color by matching it to the color of the hero slab on the section's main landing page.
+
+Breadcrumbs are not used on main section landing pages, like the following: `/about`; `/data`; `/help-for-candidates-and-committees`; `/legal-resources`; `/press`
```

Comment:
Maybe this would look better in a list format?

Reply:
Good call, I think I can make this more consistent with the formatting of other sections with a few other adjustments